eslint.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": "eslint:recommended",
```

Comment:
UI should use the jQuery config, similarly as jQuery itself does: https://github.com/jquery/jquery-ui/pull/1946

Reply:
Im not sure we can get away from using the standardized eslint configuration file